modules/core/src/main/java/org/apache/ignite/internal/processors/service/LazyServiceConfiguration.java

```diff
@@ -65,7 +68,12 @@ public LazyServiceConfiguration() {
      * @param srvcBytes Marshalled service.
      * @param interceptorsBytes Marshalled interceptors.
      */
-    public LazyServiceConfiguration(ServiceConfiguration cfg, byte[] srvcBytes, @Nullable byte[] interceptorsBytes) {
+    public LazyServiceConfiguration(
+        ServiceConfiguration cfg,
+        byte[] srvcBytes,
+        @Nullable byte[] nodeFilterBytes,
```

Comment:
`byte[] IgniteUtils#marshal(Marshaller, Object)` is not marked with `@Nullable`. `byte[] interceptorsBytes` is actually never null even if `cfg.getNodeFilter()` gives `null`.